openghg/util/_download.py

```diff
@@ -94,3 +95,4 @@ def download_data(
             else:
                 Path(filepath).write_bytes(buf.getvalue())
                 return None
+        sleep(0.1)
```

Comment:
What's the reason for this sleep?

Reply:
RichHandler has made lot of things compatible. removed sleep.